### ISSUE TITLE
theme toggle button changes only the home page background 

### ISSUE THREAD

**Issue by Subhams-GIT** [braydenidzenga/Brogram#45]

**Comment by VincenzoMeschi** [comment]
The wrong tailwind classes were being used. Also, the wrong useTheme hook was being used on the icon-cloud. 

PR: https://github.com/braydenidzenga/Brogram/pull/46

**Comment by Subhams-GIT** [comment]
Update means change signup page and fix imports?

**Comment by VincenzoMeschi** [comment]
Yes. The theme toggle works for the sign up page now. The same logic should be applied to future pages

**Comment by VincenzoMeschi** [comment]
@iamDyeus Notice that in your video, the Github logo in the icon-cloud was black while dark mode was activated. This is due to the wrong useTheme hook being imported inside the component. See line 3 of icon-cloud.jsx in your PR. It is getting useTheme from next-theme which is incorrect.

**Comment by iamDyeus** [comment]
> @iamDyeus Notice that in your video, the Github logo in the icon-cloud was black while dark mode was activated. This is due to the wrong useTheme hook being imported inside the component. See line 3 of icon-cloud.jsx in your PR. It is getting useTheme from next-theme which is incorrect.

That component was added from MagicUI using the shadcn CLI: `npx shadcn@latest add "https://magicui.design/r/icon-cloud"`

The `/getting-started` section was only intended for the initial setup. I didn't realize you guys would be using these components in the MVP as well. Now that you've pointed it out, I see that I forgot to address that detail.
